cfp.html

```diff
@@ -100,7 +100,7 @@ <h3>Call For Papers</h3>
                         <h2>Topics of Interest</h2>
                         <div class="description-one">
                             <p align="justify">
-                                VEM provides a forum where researchers and practitioners from academia and industry can present studies, tools, and data related to topics on software visualization, maintenance, and evolution.
+                                VEM provides a forum where researchers and practitioners from academia and industry can present and discuss research related to topics on software visualization, maintenance, and evolution.
                                 Topics of interest include, but are not limited to:
```

Comment:
I guess we could add a line break before the list, what do you think?

Reply:
I have added a spacing there instead of a line break.